pynta/main.py

```diff
@@ -59,15 +54,47 @@ def __init__(self,path,rxns_file,surface_type,metal,label,launchpad_path=None,fw
         self.adsorbate_fw_dict = dict()
         self.software_kwargs = software_kwargs
 
+        if software_kwargs: 
+            self.software_kwargs = software_kwargs #use user defined keywords.
+        elif self.software == 'Espresso': #user defined keywords are not provided but if software="Espresso", use keywords below.
```

Comment:
Are these defaults useful? I'm not that familiar with the nwchem inputs, but the Espresso input here isn't valid for anything except a Copper slab, it would crash with any absorbate. If we're going to have defaults the user shouldn't need to worry about the metal or the adsorbate atoms.